File: src/main/java/io/github/icusystem/icu_connect/api_icu/SettingsRequest.java
package io.github.icusystem.icu_connect.api_icu;


import java.util.ArrayList;
import java.util.List;

public class SettingsRequest {

    public String DeviceName;
    public Boolean ReactiveLED;
    public List<CameraRequest> Cameras = new ArrayList<>();

    public SettingsRequest() {

        DeviceName = "";
        ReactiveLED = true;

        CameraRequest cam = new CameraRequest();
        if (Cameras != null) {
            Cameras.add(cam);
        }
    }

}

File: src/main/java/io/github/icusystem/icu_connect/api_icu/SettingsRequest.kt
package io.github.icusystem.icu_connect.api_icu




class SettingsRequest {
     var DeviceName:String = ""
     var Cameras: MutableList<CameraRequest>? = ArrayList<CameraRequest>()

    constructor(){
        val cam = CameraRequest()
        Cameras?.add(cam)
    }




}




